Formula/libcbor.rb

```diff
@@ -13,9 +13,11 @@ class Libcbor < Formula
 
   depends_on "cmake" => :build
 
+  uses_from_macos "cjson" => :build
```

Comment:
This is a red herring (misleading). Looking at https://github.com/PJK/libcbor/blob/d115b15e9b7cbb84f0cd0f6721df0482390f6ae1/examples/CMakeLists.txt#L18-L21, cJSON is optional. The linux build failure appears to happen because of LTO/IPO.

This line and the one below it should be reverted.

Reply:
Correct, it doesn't add any functionality and will have no effect on the installed library (if it does, the build files are broken).

Reply:
@PJK Thanks for confirming that. 

@zbeekman I can't find any other uses of `uses_from_macos` for `cjson`, so I think if we keep this, it should be `depends_on`. I think the macOS build success didn't show cJSON was missing, and I didn't think to check the Jenkins logs before they were discarded.